src/reporter.ts

```diff
@@ -179,58 +204,79 @@ const generateIssueSummary = (
 
   text = summary.addHeading("Issues", 1).addEOL().addRaw(prChart).addEOL();
 
-  const totalAverageTimeToClose = calculateAverage(
-    issueMetrics.monthlyAverages.closeTime.map(([_, average]) => average),
+  const totalMedianTimeToClose = Math.round(
+    average(issueMetrics.monthlyMetrics.closeTime.map(({ median }) => median)),
+  );
+  const totalMedianTimeToFirstComment = Math.round(
+    average(
+      issueMetrics.monthlyMetrics.timeToFirstComment.map(
+        ({ median }) => median,
+      ),
+    ),
+  );
+  const totalMedianComments = Math.round(
+    average(issueMetrics.monthlyMetrics.comments.map(({ median }) => median)),
+  );
+
+  const totalAverageTimeToClose = Math.round(
+    average(
+      issueMetrics.monthlyMetrics.closeTime.map(({ average }) => average),
+    ),
   );
-  const totalAverageTimeToFirstComment = calculateAverage(
-    issueMetrics.monthlyAverages.timeToFirstComment.map(
-      ([_, average]) => average,
+  const totalAverageTimeToFirstComment = Math.round(
+    average(
+      issueMetrics.monthlyMetrics.timeToFirstComment.map(
+        ({ average }) => average,
+      ),
     ),
   );
```

Comment:
Double-aggregation introduces a lot of error! It almost never makes sense to do it.  
If you want want an average, or a median value for the whole period, calculate it directly, for all PRs/issues.